modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java

```diff
@@ -254,8 +254,36 @@ private void _sendNotification(long companyId) {
 							String body = _generateBody(
 								strAccessTokenExpirationDate);
 
-							_sendNotificationEvent(users, body);
-							_sendEmail(company, subject, body, users);
+
+
+
+							String defaultEmailFromAddress = "scim-notification@" + company.getMx();
+							String defaultEmailFromName = "SCIM-Notification";
+
+							String mail = users.get(0).getEmailAddress();
+
+							SubscriptionSender subscriptionSender = new SubscriptionSender();
+
+							//subscriptionSender.setClassName();
+							//subscriptionSender.setClassPK();
+
+							subscriptionSender.setPortletId(ScimWebKeys.SCIM_CONFIGURATION);
+
+							subscriptionSender.setEntryTitle("entryTitle");
+
+							subscriptionSender.setNotificationType(UserNotificationDefinition.NOTIFICATION_TYPE_ADD_ENTRY);
+							subscriptionSender.setBody(body);
+							subscriptionSender.setFrom(defaultEmailFromAddress, defaultEmailFromName);
+							subscriptionSender.setReplyToAddress("admin@liferay.com");
+
+							subscriptionSender.addRuntimeSubscribers("admin@liferay.com", "toName");
+
+							subscriptionSender.setMailId("popPortletPrefix", "ids");
+
+							subscriptionSender.flushNotifications();
+
+							//_sendNotificationEvent(users, body);
+							//_sendEmail(company, subject, body, users);
```

Comment:
And here also

Reply:
The comments are in previos commits but in the [final version](https://github.com/liferay-appsec/liferay-portal/blob/621ddc6205b430059cb616660b8af203fdfc6d32/modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/scheduler/ScimNotificationSchedulerJobConfiguration.java) is not commented.